packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -61,8 +61,8 @@ export interface MetricDescriptor {
   readonly unit: string;
   /** MetricDescriptor type */
   readonly type: MetricDescriptorType;
-  /** The label keys associated with the metric descriptor. */
-  readonly labelKeys: string[];
```

Comment:
The `ReadableMetric` interface is equivalent of OpenCensus [metric/v1 proto](https://github.com/census-instrumentation/opencensus-proto/blob/master/src/opencensus/proto/metrics/v1/metrics.proto#L112), we should keep `labelKeys` here as it is.